bridge_ui/src/utils/ethereum.ts

```diff
@@ -8,9 +8,36 @@ import {
 } from "../hooks/useGetSourceParsedTokenAccounts";
 import { BSC_RPC_HOST, CLUSTER, ETH_RPC_HOST, getTokenBridgeAddressForChain } from "./consts";
 import { Multicall, ContractCallContext } from 'ethereum-multicall';
+import axios from "axios"
 
 export const DefaultEVMChainConfirmations = 15
 
+interface TokenInfo {
+  address: string
+  name: string
+  symbol: string
+  decimals: number
+  logoURI: string
+}
+
+let _whitelist: TokenInfo[] | undefined = undefined
+
+async function loadETHTokenWhitelist(): Promise<TokenInfo[]> {
+  if (_whitelist !== undefined) return _whitelist
+  const { data: { tokens } } = await axios.get('https://tokens.1inch.eth.link/')
+  _whitelist = tokens
+  return tokens
+}
+
+export async function checkETHToken(tokenAddress: string) {
+  if (CLUSTER !== 'mainnet') return
+
+  const tokenWhitelist = await loadETHTokenWhitelist()
+  if (tokenWhitelist.find((token) => token.address === tokenAddress) === undefined) {
+    throw new Error(`Token ${tokenAddress} does not exists in the token list`)
```

Comment:
[Nit] `exists` -> `exist`
Does it make sense to show the link to the token list as well in the error message?

Reply:
Sounds great 👍, will update